src/main/scala-2/com/evolution/scache/CacheOpsCompat.scala

```diff
@@ -0,0 +1,100 @@
+package com.evolution.scache
+
+import cats.MonadThrow
+import cats.effect.MonadCancel
+import cats.effect.Resource
+import cats.syntax.all._
+
+import scala.util.control.NoStackTrace
+
+object CacheOpsCompat {
```

Comment:
Could you please mention the reason for the existence of this in a comment? So there would be no need to jump to this PR when someone wonders why it's needed

Reply:
done, is it good enough?

Reply:
Great, thanks!